samples/grid-lite/demo/grid-cell-format/demo.css

```
@@ -0,0 +1,60 @@
+@import url("https://code.highcharts.com/grid/css/grid.css");
+
+.custom-theme {
+    --hcg-padding: 10px;
+    --hcg-header-background: #dfdfdf;
+    --hcg-header-font-weight: 500;
+    --hcg-row-border-width: 1px;
+    --hcg-row-border-color: #d9d9d9;
+    --hcg-row-even-background: #f9f9f9;
+    --highlight-cell: #f0f2ca;
+    --high-color: #348700;
+    --low-color: #d8003a;
+    --over-30: #152fc2;
+}
+
+@media (prefers-color-scheme: dark) {
+    .custom-theme {
+        --hcg-header-background: #363636;
+        --hcg-row-border-color: #363636;
+        --hcg-row-even-background: #4f4f4f;
+        --hcg-row-background: #4f4f4f;
+        --highlight-cell: #404c2f;
+        --high-color: #98d76a;
+        --low-color: #ffa8a8;
+        --over-30: #b1befb;
+    }
+}
+
+#container {
+    margin: 8px;
```

And then you can apply background color here.
```suggestion
    padding: 8px;
    background-color: var(--hcg-background);
```